src/tron/tronUtils.ts

```diff
@@ -0,0 +1,5 @@
+// @ts-expect-error
+import { pkToAddress } from '@tronscan/client/src/utils/crypto'
```

Comment:
Add `declare module '@tronscan/client/src/utils/crypto'` to src/declare-modules.d.ts to make this come out clean.

If you want to get really fancy, you can even put full type definitions in that file to avoid this whole re-export hack.